tests/projects/failing-multi-repl-cabal-project/multi-repl-cabal-fail/multi-repl-cabal-fail.cabal

```diff
@@ -0,0 +1,23 @@
+cabal-version:       >=2.0
+name:                multi-cabal
+version:             0.1.0.0
+build-type:          Simple
+
+library
+  exposed-modules: Lib
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base >=4.10 && < 5, filepath
+  hs-source-dirs:      src
+  default-language:    Haskell2010
+  ghc-options:         +RTS -A1m -N -RTS -Wall
+
```

Comment:
```suggestion
```

Unless needed for the test, this feels like it could be confusing in the future.

Reply:
Sure let's remove it